src/frontend/apps/impress/src/features/docs/doc-table-content/components/TableContent.tsx

```diff
@@ -108,9 +108,10 @@ export const TableContent = () => {
       $align="center"
       $padding={isHover ? 'xs' : '0'}
       $justify="center"
-      $position="relative"
+      $position="fixed"
       aria-label={t('Summary')}
       $css={css`
+        right: 30px;
```

Comment:
If you find sticky is nice, you can replace by:

```suggestion
        top: 0px;
```